src/vizier/services/adaptive_export/internal/e2e/e2e_test.go

```
@@ -0,0 +1,169 @@
+		s.mu.Unlock()
+		w.WriteHeader(200)
+		_, _ = w.Write(buf.Bytes())
+	case http.MethodPost:
+		body, _ := io.ReadAll(r.Body)
+		s.mu.Lock()
+		s.insertedSQL = append(s.insertedSQL, q)
+		s.insertBody = append(s.insertBody, body)
+		s.mu.Unlock()
+		w.WriteHeader(200)
+	default:
+		http.Error(w, "method", 405)
```

golangci-lint (2.12.1)

[error] ST1013: should use constant http.StatusMethodNotAllowed instead of numeric literal 405

(staticcheck)